ush/forecast_predet.sh

```diff
@@ -134,6 +134,30 @@ common_predet(){
   # Several model components share DATA/INPUT for input data
   if [[ ! -d "${DATA}/INPUT" ]]; then mkdir -p "${DATA}/INPUT"; fi
 
+  # For CMEPS, CICE, MOM6 and WW3 determine restart writes
+  # Note FV3 has its own restart intervals  
+  cmeps_restart_interval=${restart_interval:-${FHMAX}}
+  # restart_interval = 0 implies write restart at the END of the forecast i.e. at FHMAX
+  # Convert restart interval into an explicit list for FV3
+  if (( cmeps_restart_interval == 0 )); then
+    if [[ "${DOIAU:-NO}" == "YES" ]]; then
+      CMEPS_RESTART_FH=$(( ${FHMAX} + ${half_window} ))
+    else 
+      CMEPS_RESTART_FH=("${FHMAX}")
+    fi
+  else
+    # shellcheck disable=SC2312
+    if [[ "${DOIAU:-NO}" == "YES" ]]; then
```

Comment:
I don't think SC2312 would be raised here.
```suggestion
    if [[ "${DOIAU:-NO}" == "YES" ]]; then
```

Reply:
I'll try this, but this was there in the code that I copied this from...